apps/fabric-example/metro.config.js

```diff
@@ -1,4 +1,7 @@
 const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config');
+const {
+  withReanimated,
+} = require('../../packages/react-native-reanimated/metro-config');
```

Comment:
Users will be able to import it using this path:
`react-native-reanimated/metro-config`

Reply:
Because of this:

```
> yarn start --reset-cache
error Cannot find module 'react-native-reanimated/metro-config'
Require stack:
- /Users/mateu/Work/projects/react-native-reanimated/apps/fabric-example/metro.config.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/node_modules/import-fresh/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/dist/loaders.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/dist/createExplorer.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/dist/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/metro-config/src/loadConfig.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/metro-config/src/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/utils/loadMetroConfig.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/commands/bundle/buildBundle.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/commands/bundle/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/react-native.config.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/node_modules/cosmiconfig/dist/loaders.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/node_modules/cosmiconfig/dist/defaults.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/node_modules/cosmiconfig/dist/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/build/readConfigFromDisk.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/build/loadConfig.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/build/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native-community/cli/build/commands/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native-community/cli/build/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/cli.js.
Error: Cannot find module 'react-native-reanimated/metro-config'
Require stack:
- /Users/mateu/Work/projects/react-native-reanimated/apps/fabric-example/metro.config.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/node_modules/import-fresh/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/dist/loaders.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/dist/createExplorer.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/cosmiconfig/dist/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/metro-config/src/loadConfig.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/metro-config/src/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/utils/loadMetroConfig.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/commands/bundle/buildBundle.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/commands/bundle/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native/community-cli-plugin/dist/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/react-native.config.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/node_modules/cosmiconfig/dist/loaders.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/node_modules/cosmiconfig/dist/defaults.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/node_modules/cosmiconfig/dist/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/build/readConfigFromDisk.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/build/loadConfig.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/@react-native-community/cli-config/build/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native-community/cli/build/commands/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/node_modules/@react-native-community/cli/build/index.js
- /Users/mateu/Work/projects/react-native-reanimated/node_modules/react-native/cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/mateu/Work/projects/react-native-reanimated/apps/fabric-example/metro.config.js:2:43)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
```

Reply:
It works in another project's monorepo setup, though, so I think the problem is only with path resolution in our reanimated monorepo.

Reply:
It seems that I needed to run `yarn build` and it started to work then